shared_resources/json/en.lproj/walk_section_step.json

```diff
@@ -0,0 +1,135 @@
+{
+    "identifier": "walk",
+    "type": "section",
+    "asyncActions": [{
+        "identifier": "motion",
+        "type": "motion",
+        "requiresBackgroundAudio": true,
+        "recorderTypes": ["accelerometer", "gyro", "gravity", "userAcceleration", "attitude", "rotationRate"],
+        "startStepIdentifier": "motion",
+        "stopStepIdentifier": "motion"
+    }],
+    "steps": [{
+        "identifier": "walkInstructions",
+        "type": "instruction",
+        "title": "Walk for 30 seconds",
+        "subtitle": "You will be walking continuously at your normal pace for 30 seconds. Try to walk in a straight path. If you need to turn around then turn around as if you are walking around a cone.",
+        "fullInstructionsOnly": true,
+        "image": {
+            "type": "animated",
+            "imageNames": ["walking_1", "walking_2", "walking_3", "walking_4", "walking_5", "walking_6", "walking_7", "walking_8", "walking_9", "walking_10", "walking_11", "walking_12"],
+            "placementType": "topMarginBackground",
+            "animationDuration": 6.0
+        },
+        "actions": {
+            "goForward": {
+                "type": "default",
+                "buttonTitle": "Got it"
+            }
+        }
+    },
+    {
+        "identifier": "volumeUp",
+        "type": "instruction",
+        "title": "Turn the volume up",
+        "subtitle": "Bring your phone with you and turn up your phone volume so you can hear the instructions while you are walking.",
+        "image": {
+            "type": "fetchable",
+            "imageName": "turn_up_volume",
+            "placementType": "topMarginBackground"
+        },
+        "actions": {
+            "goForward": {
+                "type": "default",
+                "buttonTitle": "The phone’s volume is up"
```

Comment:
I thought we'd moved to not using custom titles for the buttons and just using "Next"?